package.json

```diff
@@ -86,8 +86,19 @@
     "restoreMocks": true
   },
   "babel": {
+    "presets": [
+      [
+        "@babel/preset-env",
+        {
+          "targets": {
+            "node": "8.9"
+          }
+        }
+      ],
+      "@babel/preset-flow"
+    ],
     "plugins": [
-      "transform-flow-strip-types"
+      "@babel/plugin-proposal-class-properties"
```

Comment:
We weren't using class properties before, were we? Why do we need to add this?

Reply:
I've always added it as part of a standard babel setup. I'll remove it. As you said, it isn't being used.